app/models/line_item.rb

```diff
@@ -11,7 +11,7 @@ class LineItem < ApplicationRecord
   # validates :cart_id, uniqueness: { scope: product_id }
   belongs_to :order, optional: true
   belongs_to :product
```

Comment:
@avneetsng Check if we can save line_item with invalid product id(product does not exist with product id)

Reply:
No, we can't create the line_item with the invalid product_id, it will raise validation error, saying Validation Failed: Author must exist ( ActiveRecord: :RecordInvalid)